helm-frontend/public/config.js

```diff
@@ -1,4 +1,4 @@
 window.BENCHMARK_OUTPUT_BASE_URL =
-  "https://storage.googleapis.com/crfm-helm-public/lite/";
+  "https://storage.googleapis.com/crfm-helm-public/lite/benchmark_output/";
```

Comment:
The frontend still only works with the particular public release, not whatever is in `benchmark_output` as with the old frontend?

Reply:
If you use `helm-server`, you have the option of serving either the local `benchmark_output` or a public release, depending on the value of the `--output-path` [flag](https://github.com/stanford-crfm/helm/blob/81f1fdf7da8942408bedf92f6c22e3c1f6d52e87/src/helm/benchmark/server.py#L83-L89) which [overrides config.js](https://github.com/stanford-crfm/helm/blob/81f1fdf7da8942408bedf92f6c22e3c1f6d52e87/src/helm/benchmark/server.py#L22-L33).

If you don't use `helm-server` and only use `yarn serve`, it only serves the public release, which preserves the existing behavior. This may change in the future.

Reply:
could we put a comment on how this gets overridden?